provider/resources.go

```diff
@@ -57,34 +58,16 @@ func makeResource(mod string, res string) tokens.Type {
 
 // Provider returns additional overlaid schema and metadata associated with the provider..
 func Provider() tfbridge.ProviderInfo {
-	dnsimpleProv := dnsimple.Provider()
-	p := shimv2.NewProvider(dnsimpleProv)
-
 	prov := tfbridge.ProviderInfo{
-		P:           p,
+		P:           pf.ShimProvider(shim.NewProvider()),
 		Name:        "dnsimple",
 		Description: "A Pulumi package for creating and managing dnsimple cloud resources.",
 		Keywords:    []string{"pulumi", "dnsimple"},
 		License:     "Apache-2.0",
 		Homepage:    "https://pulumi.io",
 		Repository:  "https://github.com/pulumi/pulumi-dnsimple",
 		Config:      map[string]*tfbridge.SchemaInfo{},
-		GitHubOrg:   "terraform-providers",
```

Comment:
I think we should have `GitHubOrg: "dnsimple"`, since this shows up in our generated docs, not just in docs gathering. If that breaks docs generation, we don't need to block the release on it, but please open an issue in the bridge.

Reply:
Removal of this line is an effective no-op - it just removes what is otherwise a default in the bridge :/

The real issue is that the bridge uses this info to get the docs via Go import path, and unfortunately [that is still `terraform-providers`](https://github.com/dnsimple/terraform-provider-dnsimple/blob/main/go.mod#L1). So using `dnsimple` will drop all of the docs. 🙃 

The `GitHubOrg` field does several things, and two of them (somewhat) incorrectly:
- it is used to pull in docs via go mod path - this will work correctly as-is, or with saying `terraform-providers` explicitly
- it is used to add information about the upstream org to our docs (this would be incorrect here, but it is a far less impactful error than not correctly pulling in docs.
- similarly, it attempts to discover upstream releases but those are dependent on the upstream _repo_ name, not the go module name.

This will show up in our generated docs as the incorrect org, yes. However, the bridge cannot infer _both_ the github repo name and the go module name correctly from this information, and so unfortunately we need to separate areas of concern. More in https://github.com/pulumi/upgrade-provider/issues/221 - we need to explicitly add an upstream org to a provider's configuration, at least in the case of this provider.

TL;DR: this is a no-op change that doesn't affect the current way things are being done.